cmd/updatedb.go

```diff
@@ -69,8 +69,8 @@ func updateDb(cfg db.Config) error {
 		return err
 	}
 
-	fmt.Println("Seeding table with data")
-	err = db.SeedData(cfg)
+	//fmt.Println("Seeding table with data")
```

Comment:
No More seeding data as in autoscaling time first node need to create blank table during launch. Rest nodes should use the same data without any new seed. The change is to deploy the application into horizontal scalled environment.